securedrop-protocol/src/keys/source.rs

```diff
@@ -217,8 +286,8 @@ impl SourceKeyBundle {
     pub fn from_passphrase(passphrase: &[u8]) -> Self {
         use blake2::{Blake2b, Digest};
 
-        // DH key
-        let mut dh_hasher = Blake2b::<blake2::digest::typenum::U32>::new();
+        // DH-AKEM key
+        let mut dh_hasher = Blake2b::<blake2::digest::typenum::U64>::new();
```

Comment:
(todo: this is to appease cargo for the time being, will revisit the source setup from passphrase again after all the 0.3 keys stuff is in place)

Reply:
totally. also for now the passphrase isn't a passphrase, I think we can scope that out and then figure out how to handle the source passphrases (e.g. discuss stuff like #82) later